File: 7kyu/Find the divisors/Solution.java
import java.util.ArrayList;

class Divisors {

	public static String FindDivisors (int num) {
		ArrayList<Integer> res = new ArrayList<Integer>();
		
		for (int i = 2; i < num; i++) {
			if (num % i == 0) {
				res.add(i);
			}
		}
		
		return res.size() > 0 ? "" + res : "" + num + " is prime";
	}

}

File: 7kyu/Find the divisors/Solution.php
<?php

function divisors($integer) {
    $res = [];
    for ($i = 2; $i < $integer; $i++) {
        if ($integer % $i == 0) {
            array_push($res, $i);
        }
    }
    
    return count($res) > 0 ? $res : "" . $integer . " is prime";
}